03-compounds-for-both-pdf-and-html.Rmd

```diff
@@ -0,0 +1,430 @@
+$$C(u_{1}, u_{2}; \theta) = \psi^{-1}\left( \psi(u_{1}; \theta) + \psi(u_{2}; \theta); \theta \right) \tag{9}$$
+where $\psi : [0,1] \times \Theta \rightarrow [0,\infty)$ is called generator function. And $\psi^{-1}$ is defined as its pseudo-inverse:
+$$\psi^{-1}(t; \theta) =
+\begin{cases}
+    \psi^{-1}(t; \theta) & \text{if } 0 \le t \le \psi(0; \theta) \\
+    0 & \text{if } \psi(0; \theta) \le t \le \infty
+\end{cases} \tag{10}$$
+
+Note that there are many other families of Copulas, but the two Copula models that were used the most in this study are from this family.
+
+Joe Copula belongs to Archimedean copulas and has a good ability to capture the Upper Tail Dependency, Joe Copula and its generator function can be represented like the following (@triantafyllou2024):
+$$\phi(t) = -\log\left(1 - (1 - t)^\theta\right), \ \ \theta \geq 1 \tag{11}$$
+$$\phi^{-1}(t) = 1 - ((1 - e^{-t}))^{1/\theta}, \ \ \theta \geq 1 \tag{12}$$
+$$C(u_1, u_2; \theta) = 1 - \left[ (1 - u_1)^\theta+(1 - u_2)^\theta - (1 - u_1)^\theta(1 - u_2)^\theta)\right]^{1/\theta} \tag{13}$$
+Clayton Copula also belongs to the Archimedean copulas, but it is better at capturing Lower Tail Dependencies. Clayton Copula and its generator function are represented as follows (@chesneau2023):
+$$\phi(t) = \frac{1}{\theta} (t^{-\theta} - 1), \ \ \theta \in [-1, \infty) \backslash \{0\} \tag{14}$$
+$$\phi^{-1}(t) = (1 + \theta t)^{-\frac{1}{\theta}}, \ \ \theta \in [-1, \infty) \backslash \{0\} \tag{15}$$
+$$C(u_1, u_2; \theta) = \left[\max\{u_1^{-\theta} + u_2^{-\theta} -1;0\}\right]^{-\frac{1}{\theta}} \tag{16}$$
+Since in this study we focus on the Upper Tail Dependence, we do not use the Clayton Copula directly, but we can obtain the Survival Clayton Copula, also known as the rotated Clayton Copula, by rotating the Clayton Copula.
+
+### Intuitive Hazard Scenarios
+AND Scenario:
+$$ \text{P}_\text{AND} = P(U > u \cap V > v) = 1 - u - v - C(u, v) \tag{17}$$
+OR Scenario:
+$$ \text{P}_\text{OR} = P(U > u  \cup V > v) = 1 - C(u, v) \tag{18}$$
+Kendall Scenario:
+$$ \text{P}_\text{K} = P(C(U, V) > t) = 1 - K(t) \tag{19}$$
+Survival Kendall Scenario:
+$$ \text{P}_\text{SK} = P( \hat{C}(1 - U, 1 - V) < t) = \hat{K}(t) \tag{20}$$
+Of the four Intuitive Hazard Scenarios cited, $\text{P}_\text{AND}$ and $\text{P}_\text{OR}$ are very easy to understand, Pand considers both variables to be large, while Por considers that only one of the variables needs to be large. The $\text{P}_\text{K}$ and $\text{P}_\text{SK}$ can be characterized by the "critical layers" $L_t$ and $\bar{L}_t$, respectively, which separates the bivariate space into a critical region and a non-critical region (@zscheischler2020).
+
+## Descriptive Analysis
+### Climate Map of Germany
+
+```{r climatemap1, cache=FALSE, fig.cap = 'Temperature map in Germany from March to November', out.width='100%', fig.align='center', echo=FALSE, eval = TRUE}
```

Comment:
add your name here in the chunk name to avoid double chunks
e.g. climatemap1_shiyu
please also adjust for all other chunks that are named

Reply:
Have renamed all chunks to “... -shiyu”